jena-arq/src/main/java/org/apache/jena/sparql/engine/http/QueryEngineHTTP.java

```diff
@@ -135,8 +135,7 @@ private QueryEngineHTTP(String serviceURI, Query query, String queryString, Http
         this.query = query;
         this.queryString = queryString;
         this.service = serviceURI;
-        // Copy the global context to freeze it.
-        this.context = new Context(ARQ.getContext());
+        this.context = ARQ.getContext();
```

Comment:
Need to take a copy of the context to isolate it from changes to the global context that may come later.

It should be, current style:

    this.context = ARQ.getContext.copy() ;